### ISSUE TITLE
Package.getStylesheetsPath is deprecated.

### ISSUE THREAD

**Issue by mmonge** [Arcath/jekyll-atom#21]
Store package style sheets in the `styles/` directory instead of `stylesheets/` in the `jekyll` package
```
Package.getStylesheetsPath (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:460:9)
Package.getStylesheetPaths (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:471:32)
Package.loadStylesheets (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:453:38)
<unknown> (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:183:19)
Package.measure (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:163:15)
Package.load (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:177:12)
```

**Comment by peterblazejewicz** [comment]
This is already fixed but not yet released:
https://github.com/Arcath/jekyll-atom/commit/32cacd4f398ed89984e1a82fda22b533c83bd8fc

**Comment by Arcath** [comment]
version 1.0.0 will fix this (see #26)
